<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-809396K" failures="1" tests="1" name="coyni_mobile.tests.SignUpTest" time="33.727" errors="0" timestamp="2021-12-15T00:43:22 IST" skipped="0">
  <testcase classname="coyni_mobile.tests.SignUpTest" name="testFieldValidationCreateAccount" time="33.727">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at ilabs.mobile.reporting.ExtentTestManager.setFailMessageInReport(ExtentTestManager.java:72)
at coyni_mobile.utilities.CommonFunctions.validateField(CommonFunctions.java:77)
at coyni_mobile.pages.SignUpPage.validatePhoneNumber(SignUpPage.java:157)
at coyni_mobile.tests.SignUpTest.testFieldValidationCreateAccount(SignUpTest.java:128)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at ilabs.MobileFramework.TestReader.run(TestReader.java:55)
at ilabs.MobileFramework.Runner.runOneIteration(Runner.java:149)
at ilabs.MobileFramework.Runner.runRangeOfIterations(Runner.java:113)
at ilabs.MobileFramework.Runner.run(Runner.java:64)
at coyni_mobile.runner.TestRunner.main(TestRunner.java:9)
]]>
    </failure>
  </testcase> <!-- testFieldValidationCreateAccount -->
  <system-out/>
</testsuite> <!-- coyni_mobile.tests.SignUpTest -->
